packages/smooth_app/lib/cards/product_cards/smooth_product_card_found.dart

```diff
@@ -89,58 +79,54 @@ class SmoothProductCardFound extends StatelessWidget {
             ),
             padding: const EdgeInsets.all(5.0),
             child: Row(
-              mainAxisAlignment: MainAxisAlignment.start,
-              crossAxisAlignment: CrossAxisAlignment.start,
               children: <Widget>[
-                Expanded(
-                  flex: 1,
-                  child: SmoothProductImage(
-                      product: product,
-                      width: screenSize.width * 0.20,
-                      height: screenSize.width * 0.20),
-                ),
-                const SizedBox(
-                  width: 8.0,
+                SmoothProductImage(
+                  product: product,
+                  width: screenSize.width * 0.20,
+                  height: screenSize.width * 0.20,
                 ),
+                const SizedBox(width: 5),
                 Expanded(
-                  flex: 2,
-                  child: Column(
-                    mainAxisSize: MainAxisSize.max,
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: <Widget>[
-                      SizedBox(
-                        width: screenSize.width * 0.65,
-                        child: Column(
+                  child: Container(
```

Comment:
This gives a dart analysis error: `SizedBox for whitespace.`.

But without that `Container` I don't know how to make the `Column` take the full height of the `Row`, and without that the `MainAxisAlignment.spaceBetween` doesn't do the job of distributing the text across the column.

Reply:
This `Container` is currently just a `SizedBox`. If you replace one with the other, it should be no-op.

Reply:
Done